docs/blog/posts/devenv-v1.0-rewrite-in-rust.md

```diff
@@ -1,6 +1,6 @@
 ---
 draft: false
-date: 2023-03-20
+date: 2024-03-20
```

Comment:
😬. I wonder why we, as humans, struggle with the change in year. It happens pretty consistently, you'd think we would be better at handling it. I feel like March is around the time when we generally get it together and stop accidentally writing out year `n - 1`.